clients/drcachesim/tests/view_test.cpp

```diff
@@ -590,6 +591,8 @@ run_chunk_tests(void *drcontext)
 int
 main(int argc, const char *argv[])
 {
+    disable_popups();
```

Comment:
Would it be better that tests implement a `test_main()` that is invoked by `main()` defined inside test_helpers.h? This way tests won't forget to call `disable_popups()` or other test setup code we may have in future.

Reply:
You mean inside test_helpers.cpp?  And then we don't even need a header file test_helpers.h at all?  OK.

Reply:
A test could still skip setup if they defined their own main().  We could keep the header and have it #define main to something else -- but a test might not include the header.

Reply:
> but a test might not include the header.

Right, but it's easier to instruct tests to use test_main, than a possibly complex sequence of setup. I guess for now it's fine because it's just one disable_popups call.

Reply:
For suite/tests every test calls OS_INIT in main but I agree it is easier to forget that.  I will make the test_main change.

Reply:
OK I found a flaw: schedule_launcher uses _tmain.

Reply:
I'm using test_helpers.h to declare disable_popups() for use with _tmain and everything else uses test_main().

Reply:
That doesn't work though with test_helpers as a library as it adds a main symbol...and to add a test_tmain requires frontendlib.